src/test/suite/language/mongoDBService.test.ts

```diff
@@ -630,6 +654,20 @@ suite('MongoDBService Test Suite', () => {
       expect(completion).to.have.property('kind', CompletionItemKind.Reference);
     });
 
+    test('provide field reference completion in aggregate when collection is specified via getCollection', async () => {
+      testMongoDBService._cacheFields('test.collection', ['price']);
+
+      const result = await testMongoDBService.provideCompletionItems(
+        "use('test'); db.getCollection('collection').aggregate({ $match: '$p' });",
+        { line: 0, character: 67 }
```

Comment:
Looks like this is an invalid aggregation:
```suggestion
        "use('test'); db.getCollection('collection').aggregate([{ $match: '$p' }]);",
        { line: 0, character: 68 }
```

Reply:
Ah nevermind! Looks like it works to pass one stage into aggregations. TIL !